chia/full_node/block_store.py

```diff
@@ -148,7 +148,7 @@ async def get_full_block_bytes(self, header_hash: bytes32) -> Optional[bytes]:
         cached = self.block_cache.get(header_hash)
         if cached is not None:
             log.debug(f"cache hit for block {header_hash.hex()}")
-            return cached
+            return bytes(cached)
```

Comment:
I'm surprised mypy missed this

Reply:
I think because LRUCache does not have the type. We should have used a type annotation for the `cached` variable